frontend/components/droplets/lessons/lesson-renderer.tsx

```
@@ -289,6 +331,10 @@ export function LessonRenderer({
               key={i}
               block={b}
               lessonId={lesson.id}
+              dropletId={droplet.id}
+              dropletName={(droplet as any).name} // Add this - you may need to update the Droplet type
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)